content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,259 @@ the product page, you can see the following sections:
+            methods: ["GET"]
+    EOF
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).
+    Now you should see the "Bookinfo Sample" page. But there are errors `Error fetching product details`
+    and `Error fetching product reviews` on the page. These errors are expected because
+    we have not granted the `productpage` workload access to the `details` and `reviews` workloads.
+    We will fix the errors in the following steps.
+
+1. Allow access to the `details` and `reviews` workloads
+
+    The following command has the following effects:
+
+    * Creates a `details-viewer` policy which allows access with `GET` method to the `details` workload for
+      service account `cluster.local/ns/default/sa/bookinfo-productpage` (representing the `details` workload).
+
+    * Creates a `reviews-viewer` policy which allows access with `GET` method to the `reviews` workload for
+      service account `cluster.local/ns/default/sa/bookinfo-productpage` (representing the `reviews` workload).
+
+    Note that in the [setup step](#before-you-begin), we created the `bookinfo-productpage` service account
+    for the `productpage` workload. This `bookinfo-productpage` service account is the authenticated identify
+    for the `productpage` workload.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: "security.istio.io/v1beta1"
+    kind: "AuthorizationPolicy"
+    metadata:
+      name: "details-viewer"
+      namespace: default
+    spec:
+      selector:
+        matchLabels:
+          app: details
+      rules:
+      - from:
+        - source:
+            principals: ["cluster.local/ns/default/sa/bookinfo-productpage"]
+        to:
+        - operation:
+            methods: ["GET"]
+    ---
+    apiVersion: "security.istio.io/v1beta1"
+    kind: "AuthorizationPolicy"
+    metadata:
+      name: "reviews-viewer"
+      namespace: default
+    spec:
+      selector:
+        matchLabels:
+          app: reviews
+      rules:
+      - from:
+        - source:
+            principals: ["cluster.local/ns/default/sa/bookinfo-productpage"]
+        to:
+        - operation:
+            methods: ["GET"]
+    EOF
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). Now you should see the "Bookinfo Sample"
+    page with "Book Details" on the lower left part, and "Book Reviews" on the lower right part. However, in the "Book Reviews" section,
+    there is an error `Ratings service currently unavailable`. This is because "reviews" workload does not have permission to access
+    "ratings" workload. To fix this issue, you need to grant the `reviews` workload access to the `ratings` workload.
+    We will show how to do that in the next step.
+
+1. Allow access to the `ratings` workload
+
+    The following command creates a policy to allow the `reviews` workload to access the `ratings` workload. Note that in the
```

Comment:
Use the explanations to lead readers towards the next step. See a suggestion on how to do so below:
```suggestion
   Next, we need to create a policy that allows the `reviews` workload to access the `ratings` workload. 
```

Reply:
I have updated the doc with new structure and this is probably not needed.